### ISSUE TITLE
LibtorrentRasterbar --with-python: make: *** [install-recursive] Error 1

### ISSUE THREAD

**Issue by clw** [Homebrew/homebrew-core#22295]
I was trying to install libtorrent-rasterbar. I've successfully installed it before, not sure why it doesn't work now.

The exact copy-paste from command to error on Terminal:
```
$ brew install libtorrent-rasterbar --with-python
==> Installing dependencies for libtorrent-rasterbar: boost, boost-python
==> Installing libtorrent-rasterbar dependency: boost
==> Downloading https://homebrew.bintray.com/bottles/boost-1.66.0.high_sierra.bo
######################################################################## 100.0%
==> Pouring boost-1.66.0.high_sierra.bottle.tar.gz
🍺  /usr/local/Cellar/boost/1.66.0: 13,101 files, 435.3MB
==> Installing libtorrent-rasterbar dependency: boost-python
==> Downloading https://homebrew.bintray.com/bottles/boost-python-1.66.0.high_si
######################################################################## 100.0%
==> Pouring boost-python-1.66.0.high_sierra.bottle.tar.gz
🍺  /usr/local/Cellar/boost-python/1.66.0: 457 files, 17.3MB
==> Installing libtorrent-rasterbar --with-python
==> Downloading https://github.com/arvidn/libtorrent/releases/download/libtorren
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws
######################################################################## 100.0%
==> ./configure --disable-silent-rules --enable-encryption --prefix=/usr/local/C
==> make install
Last 15 lines from /Users/clw/Library/Logs/Homebrew/libtorrent-rasterbar/02.make:
web_connection_base.cpp:196:8: error: cannot initialize object parameter of type 'const libtorrent::peer_connection' with an expression of type 'const libtorrent::web_connection_base'
                if (!is_connecting() && m_server_string.empty())
                     ^~~~~~~~~~~~~
web_connection_base.cpp:198:7: error: cannot initialize object parameter of type 'const libtorrent::peer_connection' with an expression of type 'const libtorrent::web_connection_base'
                if (is_connecting()) p.flags |= peer_info::connecting;
                    ^~~~~~~~~~~~~
web_connection_base.cpp:214:3: error: cannot initialize object parameter of type 'libtorrent::peer_connection' with an expression of type 'libtorrent::web_connection_base'
                sent_bytes(0, bytes_transferred);
                ^~~~~~~~~~
1 error generated.
make[1]: *** [alert.lo] Error 1
16 errors generated.
make[1]: *** [web_connection_base.lo] Error 1
libtool: compile:  clang++ -DPACKAGE_NAME=\"libtorrent-rasterbar\" -DPACKAGE_TARNAME=\"libtorrent-rasterbar\" -DPACKAGE_VERSION=\"1.1.5\" "-DPACKAGE_STRING=\"libtorrent-rasterbar 1.1.5\"" -DPACKAGE_BUGREPORT=\"arvid@libtorrent.org\" -DPACKAGE_URL=\"http://www.libtorrent.org\" -DPACKAGE=\"libtorrent-rasterbar\" -DVERSION=\"1.1.5\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_PTHREAD=1 "-DHAVE_BOOST=/**/" "-DHAVE_BOOST_SYSTEM=/**/" "-DHAVE_BOOST_CHRONO=/**/" "-DHAVE_BOOST_RANDOM=/**/" -DHAVE_CLOCK_GETTIME=1 -DNDEBUG=1 -DTORRENT_USE_OPENSSL=1 -DHAVE_PYTHON=\"2.7\" "-DHAVE_BOOST_PYTHON=/**/" -DBOOST_ASIO_HASH_MAP_BUCKETS=1021 -DBOOST_EXCEPTION_DISABLE=1 -DBOOST_ASIO_ENABLE_CANCELIO=1 -DTORRENT_BUILDING_SHARED=1 -I. -DTORRENT_BUILDING_LIBRARY -I../include -I../ed25519/src -Os -I/usr/local/Cellar/openssl/1.0.2n/include -I/usr/local/opt/boost/include -g -O2 -ftemplate-depth=120 -Wno-format-zero-length -fvisibility=hidden -fvisibility-inlines-hidden -c announce_entry.cpp -o announce_entry.o >/dev/null 2>&1
make: *** [install-recursive] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting.html
```

So I ran the troubleshooting:
```
$ brew update
Already up-to-date.
$ brew update
Already up-to-date.
$ brew doctor
Your system is ready to brew.
```
I'm on High Sierra, so XLT and Xcode are both the latest.

Thank you!

**Comment by rakkesh** [comment]
Builds fine without errors on macOS High Sierra.

```sh
brew update
brew install libtorrent-rasterbar --with-python
```

**Comment by ilovezfs** [comment]
Yes, this is fixed in 1.1.6.

@clw please fill out the issue template completely next time.

**Comment by clw** [comment]
I checked, yes, it is fixed in 1.1.16, and the one I was trying to the death was 1.1.15, at that time `brew update` kept telling me `Already up-to-date.` I do not know when the update happened, but apparently when rakkesh posted the "fix", the update already happened. 

When I ran `brew update` after the above replies, it took a long time and a long list of items got updated, including libtorrent-rasterbar. So whoever posted the update, thank you!

I'm no stranger to homebrew, but I am no expert either.

```
$ brew update
Updated 3 taps (caskroom/cask, homebrew/core, homebrew/science).
==> New Formulae
blast                                    libtomcrypt
==> Updated Formulae
gdbm ✔                     hledger                    pick
checkstyle                 hugo                       povray
crystal-icr                jenkins                    rancher-cli
czmq                       libcddb                    re2
django-completion          libcdio                    sqlmap
exploitdb                  libtorrent-rasterbar       sslscan
get_iplayer                moreutils                  tnef
gnupg@2.0                  mpd                        ttfautohint
gopass                     mr                         vultr
gpg-agent                  nvc                        xxhash
gron                       pdf2htmlex                 zimg
grpc                       pegtl
==> Deleted Formulae
homebrew/science/matplotlib ✔
homebrew/science/a5
homebrew/science/abacas
homebrew/science/abinit
homebrew/science/abyss-explorer
homebrew/science/acado
homebrew/science/ace-corrector
homebrew/science/adam
homebrew/science/adapterremoval
homebrew/science/adol-c
homebrew/science/afra
homebrew/science/alembic
homebrew/science/alglib
homebrew/science/alien-hunter
homebrew/science/allpaths-lg
homebrew/science/alpscore
homebrew/science/amos
homebrew/science/analysis
homebrew/science/andi
homebrew/science/ann
homebrew/science/anvio
homebrew/science/apophenia
homebrew/science/aragorn
homebrew/science/arb
homebrew/science/arcs
homebrew/science/aribas
homebrew/science/arow++
homebrew/science/arrayfire
homebrew/science/art
homebrew/science/artemis
homebrew/science/ascii_plots
homebrew/science/astral
homebrew/science/astrometry-net
homebrew/science/ate-pairing
homebrew/science/atomic-pseudopotential-engine
homebrew/science/atompaw
homebrew/science/atpdec
homebrew/science/atram
homebrew/science/augustus
homebrew/science/bact
homebrew/science/bam-readcount
homebrew/science/bam2wig
homebrew/science/bamhash
homebrew/science/bamm
homebrew/science/bamutil
homebrew/science/barrnap
homebrew/science/bayestraits
homebrew/science/bbtools
homebrew/science/bcalm
homebrew/science/beagle
homebrew/science/beast2
homebrew/science/beetl
homebrew/science/bfc
homebrew/science/bioawk
homebrew/science/biobloomtools
homebrew/science/biocgal
homebrew/science/biointerchange
homebrew/science/biomake
homebrew/science/biopieces
homebrew/science/biopp
homebrew/science/bitseq
homebrew/science/blasr
homebrew/science/blat
homebrew/science/blaze-lib
homebrew/science/bless
homebrew/science/blis
homebrew/science/boost-compute
homebrew/science/bowtie
homebrew/science/bpel2owfn
homebrew/science/bpipe
homebrew/science/breseq
homebrew/science/busco
homebrew/science/butterflow
homebrew/science/bwtdisk
homebrew/science/calculix-ccx
homebrew/science/cantera
homebrew/science/canu
homebrew/science/cap3
homebrew/science/ccfits
homebrew/science/cd-hit
homebrew/science/cddlib
homebrew/science/cdo
homebrew/science/cdsclient
homebrew/science/cegma
homebrew/science/celera-assembler
homebrew/science/centrifuge
homebrew/science/cerulean
homebrew/science/cgns
homebrew/science/circlator
homebrew/science/circos
homebrew/science/ckon
homebrew/science/clark
homebrew/science/clips
homebrew/science/clonalframeml
homebrew/science/clonehd
homebrew/science/clustal-omega
homebrew/science/clustal-w
homebrew/science/cmdstan
homebrew/science/cmor
homebrew/science/coinmp
homebrew/science/colpack
homebrew/science/concorde
homebrew/science/corset
homebrew/science/crfsuite
homebrew/science/crlibm
homebrew/science/cryptoverif
homebrew/science/cuba
homebrew/science/cube
homebrew/science/cufflinks
homebrew/science/cusp
homebrew/science/cutadapt
homebrew/science/cvblob
homebrew/science/cytoscape
homebrew/science/dadadodo
homebrew/science/daligner
homebrew/science/data-science-toolbox
homebrew/science/dazz_db
homebrew/science/dealii
homebrew/science/deeplearning4j-cli
homebrew/science/delly
homebrew/science/des
homebrew/science/dextractor
homebrew/science/dgtal
homebrew/science/dida
homebrew/science/discovar
homebrew/science/discovardenovo
homebrew/science/dl_poly_classic
homebrew/science/dotwrp
homebrew/science/ds9
homebrew/science/dsdp
homebrew/science/dsk
homebrew/science/dssp
homebrew/science/dwgsim
homebrew/science/dynare
homebrew/science/e-mem
homebrew/science/ea-utils
homebrew/science/edena
homebrew/science/edirect
homebrew/science/einspline
homebrew/science/elemental
homebrew/science/elph
homebrew/science/emboss
homebrew/science/enblend-enfuse
homebrew/science/ensembl-tools
homebrew/science/ess
homebrew/science/etsf_io
homebrew/science/exabayes
homebrew/science/exonerate
homebrew/science/express
homebrew/science/fann
homebrew/science/fasta
homebrew/science/fastml
homebrew/science/fastq-tools
homebrew/science/fasttree
homebrew/science/fastuniq
homebrew/science/fastx_toolkit
homebrew/science/fcgene
homebrew/science/fermi
homebrew/science/fermi-lite
homebrew/science/fermi2
homebrew/science/fermikit
homebrew/science/fgsl
homebrew/science/flash
homebrew/science/flexbar
homebrew/science/flint
homebrew/science/flux-simulator
homebrew/science/fplll
homebrew/science/fqzcomp
homebrew/science/freebayes
homebrew/science/freec
homebrew/science/fsa
homebrew/science/fwdpp
homebrew/science/g2o
homebrew/science/gaemr
homebrew/science/galfit
homebrew/science/galib
homebrew/science/galsim
homebrew/science/gap
homebrew/science/garli
homebrew/science/gatb
homebrew/science/gatk
homebrew/science/geant4
homebrew/science/geda-gaf
homebrew/science/geneid
homebrew/science/genet
homebrew/science/genewise
homebrew/science/getdp
homebrew/science/gfan
homebrew/science/ggobi
homebrew/science/giira
homebrew/science/gingr
homebrew/science/glimmer3
homebrew/science/glimmerhmm
homebrew/science/glpk448
homebrew/science/gmap-gsnap
homebrew/science/gmcloser
homebrew/science/gmtk
homebrew/science/gnuastro
homebrew/science/gnudatalanguage
homebrew/science/grabix
homebrew/science/graph-tool
homebrew/science/graphlan
homebrew/science/gtsam
homebrew/science/gubbins
homebrew/science/h5utils
homebrew/science/harry
homebrew/science/harvest-tools
homebrew/science/hdf4
homebrew/science/healpix
homebrew/science/hisat
homebrew/science/hisat2
homebrew/science/hlaminer
homebrew/science/hmmer
homebrew/science/hmmer2
homebrew/science/hopdm
homebrew/science/hotspot
homebrew/science/htsbox
homebrew/science/humann2
homebrew/science/hyphy
homebrew/science/idba
homebrew/science/idcoefs
homebrew/science/igvtools
homebrew/science/impute2
homebrew/science/infernal
homebrew/science/insighttoolkit
homebrew/science/inter-views
homebrew/science/ipopt
homebrew/science/iqtree
homebrew/science/itensor
homebrew/science/itsol
homebrew/science/iva
homebrew/science/jblas
homebrew/science/jellyfish
homebrew/science/jellyfish-1.1
homebrew/science/jmol
homebrew/science/joinx
homebrew/science/jsbsim
homebrew/science/k8
homebrew/science/kaiju
homebrew/science/kalign
homebrew/science/kat
homebrew/science/kent-tools
homebrew/science/kissplice
homebrew/science/kmacs
homebrew/science/kmc
homebrew/science/kmergenie
homebrew/science/kmerstream
homebrew/science/kollector
homebrew/science/kraken
homebrew/science/lapack-manpages
homebrew/science/last
homebrew/science/lastz
homebrew/science/libbi
homebrew/science/libbigwig
homebrew/science/libbuddy
homebrew/science/libccd
homebrew/science/libcerf
homebrew/science/libctl
homebrew/science/libdivsufsort
homebrew/science/libeemd
homebrew/science/libfolia
homebrew/science/libgpkg
homebrew/science/liblbfgs
homebrew/science/libminc
homebrew/science/libpll
homebrew/science/libsbml
homebrew/science/libsbol
homebrew/science/libsequence
homebrew/science/libsigrokdecode
homebrew/science/lie
homebrew/science/lighter
homebrew/science/lightstringgraph
homebrew/science/links-scaffolder
homebrew/science/lis
homebrew/science/littler
homebrew/science/lmfit
homebrew/science/lmod
homebrew/science/lobstr
homebrew/science/lola
homebrew/science/lp_solve
homebrew/science/lrsim
homebrew/science/lsd
homebrew/science/lumpy-sv
homebrew/science/m4ri
homebrew/science/macse
homebrew/science/mad-x
homebrew/science/madlib
homebrew/science/mafft
homebrew/science/maker
homebrew/science/mallet
homebrew/science/mantaflow
homebrew/science/mapsembler2
homebrew/science/maq
homebrew/science/mash
homebrew/science/masurca
homebrew/science/mathgl
homebrew/science/mathomatic
homebrew/science/maude
homebrew/science/mcl
homebrew/science/med-file
homebrew/science/megahit
homebrew/science/megam
homebrew/science/meme
homebrew/science/meraculous
homebrew/science/metaphlan
homebrew/science/methpipe
homebrew/science/metis4
homebrew/science/mfem
homebrew/science/mfusg
homebrew/science/mhap
homebrew/science/minced
homebrew/science/minia
homebrew/science/miniasm
homebrew/science/minimap
homebrew/science/mir-prefer
homebrew/science/mira
homebrew/science/mitofy
homebrew/science/mlpack
homebrew/science/mlst
homebrew/science/moab
homebrew/science/molden
homebrew/science/mothur
homebrew/science/mpsolve
homebrew/science/mrbayes
homebrew/science/mrfast
homebrew/science/msieve
homebrew/science/mtl
homebrew/science/multi-worm-tracker
homebrew/science/mummer
homebrew/science/mumps
homebrew/science/muscle
homebrew/science/nanoflann
homebrew/science/nanopolish
homebrew/science/nauty
homebrew/science/ncbi-c++-toolkit
homebrew/science/nccmp
homebrew/science/ncl
homebrew/science/nest
homebrew/science/neuron
homebrew/science/newick-utils
homebrew/science/newicktools
homebrew/science/nextflow
homebrew/science/nexusformat
homebrew/science/nfft
homebrew/science/nglib
homebrew/science/niftilib
homebrew/science/nip2
homebrew/science/nixio
homebrew/science/nonpareil
homebrew/science/novoalign
homebrew/science/ntcard
homebrew/science/numdiff
homebrew/science/nusmv
homebrew/science/nxtrim
homebrew/science/oases
homebrew/science/oce
homebrew/science/ococo
homebrew/science/ogdraw
homebrew/science/oma
homebrew/science/omcompiler
homebrew/science/ome-common
homebrew/science/ome-files
homebrew/science/ome-xml
homebrew/science/openalpr
homebrew/science/openbr
homebrew/science/opencollada
homebrew/science/openfst
homebrew/science/opengrm-ngram
homebrew/science/opengrm-thrax
homebrew/science/openimageio
homebrew/science/openmeeg
homebrew/science/openni
homebrew/science/openni2
homebrew/science/orocos-kdl
homebrew/science/orthofinder
homebrew/science/osgearth
homebrew/science/oswitch
homebrew/science/p4est
homebrew/science/paml
homebrew/science/pandaseq
homebrew/science/parallel-netcdf
homebrew/science/paraview
homebrew/science/parmetis
homebrew/science/parsnp
homebrew/science/pastix
homebrew/science/pathd8
homebrew/science/pathvisio
homebrew/science/paxtools
homebrew/science/pbsuite
homebrew/science/pcap
homebrew/science/pear
homebrew/science/perf
homebrew/science/petsc
homebrew/science/phipack
homebrew/science/phlawd
homebrew/science/phylip
homebrew/science/phyml
homebrew/science/phyutility
homebrew/science/phyx
homebrew/science/piler
homebrew/science/pilercr
homebrew/science/pilon
homebrew/science/plasma
homebrew/science/platypusvar
homebrew/science/plink
homebrew/science/plink2
homebrew/science/plr
homebrew/science/pnapi
homebrew/science/poa
homebrew/science/pocl
homebrew/science/populations
homebrew/science/poretools
homebrew/science/prank
homebrew/science/primer3
homebrew/science/prokka
homebrew/science/prooftree
homebrew/science/proteinortho
homebrew/science/proverif
homebrew/science/psmc
homebrew/science/pspp
homebrew/science/pulseview
homebrew/science/pykep
homebrew/science/qcl
homebrew/science/qr_mumps
homebrew/science/qsopt
homebrew/science/qsopt_ex
homebrew/science/quaff
homebrew/science/quake
homebrew/science/qualimap
homebrew/science/quast
homebrew/science/quest
homebrew/science/quicktree
homebrew/science/quip
homebrew/science/quorum
homebrew/science/r8s
homebrew/science/racon
homebrew/science/radx
homebrew/science/rainbow
homebrew/science/rampart
homebrew/science/rapsearch2
homebrew/science/rate4site
homebrew/science/raxml
homebrew/science/ray
homebrew/science/rcorrector
homebrew/science/readseq
homebrew/science/readsim
homebrew/science/reapr
homebrew/science/recon
homebrew/science/repeatmasker
homebrew/science/repeatmodeler
homebrew/science/repeatscout
homebrew/science/rmblast
homebrew/science/rml-mmc
homebrew/science/rna-star
homebrew/science/rnammer
homebrew/science/ropebwt
homebrew/science/ropebwt2
homebrew/science/sailfish
homebrew/science/sais
homebrew/science/sally
homebrew/science/salmon
homebrew/science/salt
homebrew/science/sambamba
homebrew/science/samblaster
homebrew/science/samtools@0.1
homebrew/science/sara
homebrew/science/sbagen
homebrew/science/scamp
homebrew/science/scarpa
homebrew/science/scotch
homebrew/science/scotch5
homebrew/science/scram
homebrew/science/scrm
homebrew/science/sdsl-lite
homebrew/science/seq-gen
homebrew/science/seqan
homebrew/science/seqdb
homebrew/science/sequel
homebrew/science/sextractor
homebrew/science/sfscode
homebrew/science/sga
homebrew/science/shark
homebrew/science/shogun
homebrew/science/shrimp
homebrew/science/sickle
homebrew/science/sigrok-cli
homebrew/science/silo
homebrew/science/simpleitk
homebrew/science/simulate-pcr
homebrew/science/siril
homebrew/science/sisl
homebrew/science/skewer
homebrew/science/slepc
homebrew/science/slicot
homebrew/science/sllib
homebrew/science/smalt
homebrew/science/smrtanalysis
homebrew/science/snap
homebrew/science/snap-aligner
homebrew/science/snid
homebrew/science/snoscan
homebrew/science/snp-sites
homebrew/science/snpeff
homebrew/science/soapdenovo
homebrew/science/sollya
homebrew/science/soplex
homebrew/science/sortmerna
homebrew/science/spaced
homebrew/science/spatialite-gis
homebrew/science/spici
homebrew/science/squeezambler
homebrew/science/ssake
homebrew/science/stacks
homebrew/science/stiff
homebrew/science/stringtie
homebrew/science/sumo
homebrew/science/superlu43
homebrew/science/superlu_dist
homebrew/science/superlu_mt
homebrew/science/swarm
homebrew/science/swetest
homebrew/science/swrcfit
homebrew/science/symengine
homebrew/science/symphony
homebrew/science/t-coffee
homebrew/science/tabtk
homebrew/science/tagdust
homebrew/science/tamarin-prover
homebrew/science/tasr
homebrew/science/tbl2asn
homebrew/science/tetgen
homebrew/science/therion
homebrew/science/ticcutils
homebrew/science/timbl
homebrew/science/tisean
homebrew/science/tmv-cpp
homebrew/science/topcat
homebrew/science/tophat
homebrew/science/trans-abyss
homebrew/science/trans_proteomic_pipeline
homebrew/science/transdecoder
homebrew/science/transpose
homebrew/science/transrate-tools
homebrew/science/transtermhp
homebrew/science/trf
homebrew/science/triangle
homebrew/science/trilinos
homebrew/science/trimadap
homebrew/science/trimal
homebrew/science/trimmomatic
homebrew/science/trinity
homebrew/science/trnascan
homebrew/science/ucsc-genome-browser
homebrew/science/ucto
homebrew/science/unafold
homebrew/science/unicycler
homebrew/science/uniqtag
homebrew/science/uproc
homebrew/science/utgb
homebrew/science/vague
homebrew/science/varscan
homebrew/science/vcake
homebrew/science/vcfanno
homebrew/science/vcflib
homebrew/science/velvet
homebrew/science/velvetoptimiser
homebrew/science/viennarna
homebrew/science/vigra
homebrew/science/vislcg3
homebrew/science/visp
homebrew/science/vsearch
homebrew/science/vt
homebrew/science/wcalc
homebrew/science/wcslib
homebrew/science/wcstools
homebrew/science/weblogo
homebrew/science/wfdb
homebrew/science/wiggletools
homebrew/science/wopr
homebrew/science/xbyak
homebrew/science/xcdf
homebrew/science/xfig
homebrew/science/xmgredit
homebrew/science/xmi-msim
homebrew/science/xraylib
homebrew/science/xrmc
homebrew/science/xylib
homebrew/science/yaggo
homebrew/science/yaha
homebrew/science/yass
homebrew/science/yeppp
homebrew/science/yices
homebrew/science/zoltan
```
After that, libtorrent-rasterbar installed fine.

**Comment by ilovezfs** [comment]
>So whoever posted the update, thank you!

You're welcome.
https://github.com/Homebrew/homebrew-core/pull/22314
